CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @molawson @sheck @danielma
```

Comment:
In light of the recent creation of a "Balto Maintainers" team:

```suggestion
* @planningcenter/balto-maintainers
```

Reply:
Yep... This looks like the route to go 👆🏼